ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityEmailQuestionBlock.ts

```diff
@@ -0,0 +1,23 @@
+import { ActivityTextQuestionBlock } from './activityTextQuestionBlock';
+
+export class ActivityEmailQuestionBlock extends ActivityTextQuestionBlock {
+  public email: string | null = null;
+  public emailConfirmation: string | null = null;
+  constructor() {
```

Comment:
This is one of the weirdnesses that needed to figure out with this component. For all others we have had just one value that we care about (granted ,that value could be an array). Here server only cares about the email, but client cares about confirmation value as well for validation purposes, so had to get creative to fit into our existing scheme of one block/one value.